ui/perfherder/graphs/LegendCard.jsx

```diff
@@ -1,13 +1,14 @@
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 /* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable prefer-destructuring */
```

Comment:
This was annoyingly hit because I used `item.color[1]`... I can do the destructuring assignment but it seems a bit silly here since it won't be reused.